src/storage/mod.rs

```diff
@@ -461,15 +461,15 @@ impl<T> PackedData<T> {
     }
 }
 
-/// Used by the framework to quickly join componets
+/// Used by the framework to quickly join components.
 pub trait UnprotectedStorage<T>: Debug + Sized {
     /// Creates a new `Storage<T>`. This is called when you register a new
     /// component type within the world.
     fn new() -> Self;
 
     /// Clean the storage given a check to figure out if an index
     /// is valid or not. Allows us to safely drop the storage.
-    unsafe fn clean<F>(&mut self, F) where F: Fn(Index) -> bool;
+    unsafe fn clean<F>(&mut self, f: F) where F: Fn(Index) -> bool;
```

Comment:
I personally use `f` to accept closures very often. I also saw it in other libraries, and in case somebody doesn't know, the trait bound isn't far